src/builddef.c

```diff
@@ -136,6 +136,7 @@ int main(int argc, char **argv) {
     envar_set("PATH", "/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin", 1);
     envar_set("SINGULARITY_ROOTFS", CONTAINER_MOUNTDIR, 1);
     envar_set("SINGULARITY_libexecdir", singularity_registry_get("LIBEXECDIR"), 1);
+    envar_set("SINGULARITY_bindir", singularity_registry_get("BINDIR"), 1);
```

Comment:
We could do this another way too... If you look at src/Makefile.am, you may notice the AM_CPPFLAGS make it possible to define this statically to be included within the source file (e.g. can't be externally modified). When we spoke about this earlier, I didn't consider making it completely static, but I think now it would be a good idea. Let's do that instead of setting this through the registry (which can be modified).

Reply:
OK.  I've tried a bunch of stuff, but I'm not seeing how to do this (although it seems like it should be trivial).  Can you give me another hint?

Reply:
This isn't really a hint, more so an implementation, but check out https://github.com/singularityware/singularity/commit/49465e51256625d3cd719237503481f9eb028867. Hope that helps for next time!